http/src/test/java/com/github/awsjavakit/http/ParameterStoreCachedTokenProviderTest.java

```diff
@@ -90,15 +93,14 @@ void shouldReuseTokenStoredInSsmWhenTokenExistsInSsmAndIsValid() {
     server.verify(exactly(1), postRequestedFor(urlPathEqualTo(AUTH_PATH.toString())));
     assertThat(ssmClient.getReadCounter().get()).isEqualTo(READ_TWICE_ON_FAILURE_ONCE_IN_SUCCESS);
     assertThat(ssmClient.getWriteCounter().get()).isEqualTo(WRITE_ON_FAILURE);
-    assertThat(actualToken).isEqualTo(this.accessToken);
+    assertThat(actualToken.value()).isEqualTo(this.accessToken);
   }
 
   @Test
-  void shouldAssumeThatSomeoneElseIsTryingToGenerateTheTokenWhenTokenIsNotValid() {
+  void shouldAssumeThatCollisionDuringUpdateIsLikelyToHappen() {
```

Comment:
the implementation of this test is a quick fix to avoid writing the parameter too many times.